### ISSUE TITLE
Can't install zc.recipe.egg using pip

### ISSUE THREAD

**Issue by nazrulworld** [buildout/buildout#404]
We have a recipe https://pypi.python.org/pypi/plone.recipe.sublimetext/ that is dependent on  `zc.recipe.egg`. When I tried to install using pip it fails, Lastly I find out problem is in `zc.recipe.egg` that can't be installed through pip.
Bellows are details

PIP Version: 9.0.1
Python: 2.7.12

`Exception:
Traceback (most recent call last):
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/commands/install.py", line 335, in run
    wb.build(autobuilding=True)
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/wheel.py", line 749, in build
    self.requirement_set.prepare_files(self.finder)
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/req/req_set.py", line 380, in prepare_files
    ignore_dependencies=self.ignore_dependencies))
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/req/req_set.py", line 666, in _prepare_file
    check_dist_requires_python(dist)
  File "/tmp/py2/local/lib/python2.7/site-packages/pip/utils/packaging.py", line 48, in check_dist_requires_python
    feed_parser.feed(metadata)
  File "/usr/lib/python2.7/email/feedparser.py", line 177, in feed
    self._input.push(data)
  File "/usr/lib/python2.7/email/feedparser.py", line 99, in push
    parts = data.splitlines(True)
AttributeError: 'NoneType' object has no attribute 'splitlines'
`

**Comment by jamadden** [comment]
This is actually a limitation in zc.recipe.egg...it assumes anything that ends in ".egg" is a Python egg file, which is obviously not the case. 

The solution is pip's `--no-binary` argument:
```
pip install --no-binary=zc.recipe.egg zc.recipe.egg # or whatever depends on zc.recipe.egg
```

That can go in a `requirements.txt` too, of course.

**Comment by jamadden** [comment]
Pip bug: https://github.com/pypa/pip/issues/3028

**Comment by nazrulworld** [comment]
Thank you so much @jamadden

**Comment by leorochael** [comment]
> We have a recipe https://pypi.python.org/pypi/plone.recipe.sublimetext/ that is dependent on `zc.recipe.egg`. When I tried to install using pip it fails, Lastly I find out problem is in `zc.recipe.egg` that can't be installed through pip.

@nazrulworld, why do you need to install it with pip? Isn't it better to let buildout itself install it?

In any case, the underlying bug, which is pypa/setuptools#462, has a proposed fix on pypa/setuptools#1116.
